client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -939,6 +939,9 @@ Params withIndicesOptions(IndicesOptions indicesOptions) {
                     expandWildcards = joiner.toString();
                 }
                 putParam("expand_wildcards", expandWildcards);
+                if(!indicesOptions.ignoreThrottled()){
```

Comment:
I think the `ignore_throttled` should always be serialized in the context here:

```
putParam("ignore_throttled", Boolean.toString(indicesOptions.ignoreThrottled()));
```

Whether `ignore_throttled` is enabled or disabled by default depends on the api, and the `withIndicesOptions (...)` method is used for multiple APIs.

Reply:
Yes I agree with your changes. I need to do something more ? Sorry I’m newbie collaborating in open source

Reply:
Thanks for your help! I agree with your changes and I already pushed these changes